numba/targets/arrayobj.py

```diff
@@ -1839,6 +1839,30 @@ def np_unique_impl(a):
     return np_unique_impl
 
 
+@overload(np.repeat)
+def np_repeat(a, repeats):
+    if not isinstance(a, (types.Array,
+                          types.Sequence,
```

Comment:
Is `Sequence` needed/supported ? Try passing `somearray.flat` into this implementation.

Reply:
`somearray.flat` fails

Reply:
So `somearray.flat` fails because that returns a `flatiter` which is not accepted the same way `iter(array)` and `range(10)` aren't. If I remove `Sequence` however, the `list` type is no longer accepted, which would be bad.

Reply:
Alright, I decided to go with `List` and `BaseTuple` instead of `Sequence`.